### PR TITLE
SAK-43613 Samigo: Modify text of assessment preview banner

### PR DESCRIPTION
**Pull request by MRutea**
https://jira.sakaiproject.org/browse/SAK-43613

### CODE REVIEW

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages_de_DE.properties [hunk 1]
```diff
@@ -7,7 +7,10 @@ add_q=Frage hinzuf\u00fcgen\:
 addr=Adressen\:
 admin=Admin
 done=Fertig
-ass_preview=Pr\u00fcfungsvorschau - Beispieldarstellung, wie Studierenden diese Pr\u00fcfung angezeigt wird                                                            
+exit_preview=Exit Preview
+ass_preview_1=Assessment Preview - This preview provides an approximate view of the layout of the assessment for students.
+ass_preview_2= The preview will <strong>not</strong> allow you to save or submit answers, display the timer if there is a time limit, or access any feedback.
+ass_preview_3= The preview also does <strong>not</strong> take into account students' ability (or lack thereof) to view images or other media embedded in questions.
```
Comment (adrianfish) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages_de_DE.properties:13]: You don't need to add all these keys in each language. Just add them in the base properties file.

**Review by adrianfish — CHANGES_REQUESTED** [review]

**Comment by MRutea** [comment]
@adrianfish 
Thank you, Adrian.
I've made the correction.
Sorry.

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties [hunk 2]
```diff
@@ -7,7 +7,10 @@ add_q=Add Question:
 addr=Addresses:
 admin=Admin
 done=done
-ass_preview=Assessment Preview - This is an example student view of this assessment                                                            
+exit_preview=Exit Preview
+ass_preview_1=Assessment Preview - This preview provides an approximate view of the layout of the assessment for students.
+ass_preview_2= The preview will <strong>not</strong> allow you to save or submit answers, display the timer if there is a time limit, or access any feedback.
+ass_preview_3= The preview also does <strong>not</strong> take into account students' ability (or lack thereof) to view images or other media embedded in questions.
```
Comment (adrianfish) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties:13]: You don't actually need to put these strings into 3 properties. You can do multiline properties in Java, just use a backslash at the end of each line, just for readability. That way the whole thing is loaded up into one element during the faces render. So, I'd just update ass_preview with whole suggested text, with those backslashes on the end of the lines.

**Review by adrianfish — CHANGES_REQUESTED** [review]

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties (hunk 2 above)
Comment (MRutea) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties]: Hello, Adrian;
@adrianfish 
Thank again for the suggestions. 
Now there's a cleaner commit.

Greetings.

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties [hunk 3]
```diff
@@ -7,7 +7,8 @@ add_q=Add Question:
 addr=Addresses:
 admin=Admin
 done=done
-ass_preview=Assessment Preview - This is an example student view of this assessment                                                            
+exit_preview=Exit Preview
+ass_preview=Assessment Preview - This preview provides an approximate view of the layout of the assessment for students. \The preview will <strong>not</strong> allow you to save or submit answers, display the timer if there is a time limit, or access any feedback. \The preview also does <strong>not</strong> take into account students' ability (or lack thereof) to view images or other media embedded in questions.
```
Comment (fostersdesign) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties:11]: Are the slashes needed at the beginning of each sentence? In your screenshots on the Jira issue, the sentences are missing a single space in between period and the start of the next sentence.

Comment (fostersdesign) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties:11]: I suggest we modify the last sentence to:

`The preview does <strong>not</strong> reflect whether students can access or view images or other media embedded in questions.`

**Review by fostersdesign — CHANGES_REQUESTED** [review]

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties (hunk 3 above)
Comment (MRutea) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties]: In first place, thank you for your comments.
It was only for readability, given that there was three sentences in a row.

Comment (MRutea) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties]: Done!

Comment (fostersdesign) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties]: Thanks for making those changes, @MRutea.

**Comment by fostersdesign** [comment]
Tiffany has some questions and comments on the Jira issue about the latest screenshot.

**Review by adrianfish — APPROVED** [review]
GTG.

File: library/src/morpheus-master/sass/modules/tool/samigo/_samigo.scss [hunk 4]
```diff
@@ -421,4 +421,9 @@
 		display: none;
 	}
 
+	.exitPreviewButton {
+		margin-top: 5px;
+		margin-left: 80px;
+	}
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/tool/samigo/_samigo.scss:427]: The spacing here seems a bit arbitrary. The button should really be right-aligned, but more work on the banners needs to be done first (which I'm working on for another PR). 

For now, I think a standard space between the end of the text and the button will be sufficient. 

```suggestion
	.exitPreviewButton {
		margin-left: $standard-space;
	}
```

**Review by fostersdesign — APPROVED** [review]
Thanks @MRutea for making those changes for us. Looks pretty good now. I've just left one code suggestion about the spacing of the button. Thanks.

File: library/src/morpheus-master/sass/modules/tool/samigo/_samigo.scss (hunk 4 above)
Comment (MRutea) [reply library/src/morpheus-master/sass/modules/tool/samigo/_samigo.scss]: Thank you, @fostersdesign. I'v made this change that you suggested and made a new commit.
Greetings.

Comment (fostersdesign) [reply library/src/morpheus-master/sass/modules/tool/samigo/_samigo.scss]: Thanks.

**Pull request merged**